CHANGELOG.rst

```diff
@@ -19,8 +19,76 @@
   ``mutation.index`` will still work through deprecated aliases,
   but new code should use a more cite-centric approach.
 
+- The ``TreeSequence.diffs()`` method no longer works. Please use
+  the ``TreeSequence.edge_diffs()`` method instead.
 
-**Deprecated aliases**:
+- TreeSequence.get_num_records() no longer works. Any code using
+  this or the records() iterator should be recast to work with
+  the edges() iterator and num_edges instead.
+
+
+**New features**:
+
+- The API has been made more Pythonic by replacing (e.g.)
+  ``tree.get_parent(u)`` with ``tree.parent(u)``, and
+  ``tree.get_total_branch_length()`` with ``tree.total_branch_length``.
+  The old forms have been maintained as deprecated aliases. (#64)
+
+- Support for a much more general class of tree sequence topologies.
+  For example, trees with multiple roots are fully supported.
+
+- Substantially generalised mutation model. Mutations now occur at
+  specific sites, which can be associated with zero to many mutations.
+  Each site has an ancestral state (any character string) and
+  each mutation a derived_state (any character string).
+
+- The variants() method now returns a list of alleles for each
+  site, and genotypes are indexes into this array. This is both
+  consistent with existing usage, and generalisable to any character
+  strings used as mutational states.
```

Comment:
I find the last clause a bit unclear. But I might misunderstand something. Would subbing this for L47-L48 still say the same thing? 

```
consistent with existing usage and works with the newly generalised mutation model,
which allows arbitrary strings of characters as mutational states.
```

If so I find this wording clearer.